ceph-pr-docs/build/build

```diff
@@ -12,4 +12,34 @@ sudo rm -f /etc/apt/sources.list.d/ceph-boost*
 sudo apt-get update -o Acquire::Languages=none -o Acquire::Translation=none
 sudo apt-get install -y gcc python3-dev python3-pip python3-virtualenv libxml2-dev libxslt-dev doxygen graphviz ant ditaa cython3
 
-timeout 3600 ./admin/build-doc
+virtualenv -q --python python3 venv
+. venv/bin/activate
+pip install tox
+pip install git+https://github.com/ceph/githubcheck.git
+sha1=$(git rev-parse refs/remotes/origin/pr/${ghprbPullId}/head)
```

Comment:
[`GIT_COMMIT`](https://plugins.jenkins.io/git/#environment-variables)?

Reply:
i think `GIT_COMMIT` is the *merge* commit, not the one of the sha1 of the change itself. and it somehow fails to serve us sometimes. see 74d8f325149969a7527355c9df2c40b62c354cf4 .

Reply:
Weird :s I just saw a few related bugs on Jenkins Jira...